src/styles/blocks/card-block/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars-N {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:nth-child(-n+1),
+  &__star:nth-child(-n+2),
+  &__star:nth-child(-n+3),
+  &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+
```

Comment:
<img width="1060" alt="Screenshot 2023-06-29 at 18 12 36" src="https://github.com/mate-academy/layout_catalog/assets/135232001/f06c735b-b180-4802-b395-1d7bd6c2375c">
Use the checklist from the task to check your code